content/support-training.html

```diff
@@ -49,6 +49,12 @@ <h2>Courses</h2>
               <img src="../assets/commercial-support-logos/robust-perception.png" alt="Robust Perception">
             </a>
           </div>
+
+          <div class="col-md-3 col-sm-6 doc-content tc-col">
```

Comment:
We currently have a rule to keep lists like these alphabetically sorted, which would put you at the front of the list. Could you update the ordering accordingly?